### ISSUE TITLE
debugger terminates a Rails console when you type `n`

### ISSUE THREAD

**Issue by dblock** [cldwalker/debugger#133]
Via https://github.com/nixme/pry-debugger/issues/55.

```
source 'https://rubygems.org'

ruby '2.0.0'

gem 'rails'
gem 'sqlite3'
gem 'debugger'
```

```
$ rails c
Loading development environment (Rails 4.1.4)
2.0.0-p353 :001 > n
/tmp/n-wat$ 
```

This deserved an animated GIF.

![rails-c](https://cloud.githubusercontent.com/assets/542335/3499032/8c225826-05fb-11e4-8cb8-4f1c87079c74.gif)

It's either confusing or a bug?

The `n` method is defined via `ExtendCommandBundle.def_extend_command "n", :Next` in https://github.com/cldwalker/debugger/blob/master/lib/ruby-debug/commands/irb.rb#L22 and is implemented as 

``` ruby
    class Next # :nodoc:
      def self.execute(conf)
        throw :IRB_EXIT, :next
      end
    end
```

I really don't understand the purpose of this.

**Comment by cldwalker** [comment]
Nice gif! This is old behavior back from when debugger was on rubyforge, yikes. Given that [debugger will soon be end-of-lifed](https://github.com/cldwalker/debugger#status), I won't be taking any enhancements to implement next. Feel free to close if you'd like

**Comment by dblock** [comment]
Makes sense, thanks @cldwalker - will leave this open since it's still an issue, eventually the project will be lost in the Github depths.
